sympy/printing/ccode.py

```diff
@@ -231,6 +231,19 @@ def _print_Symbol(self, expr):
         else:
             return name
 
+    def _print_Relational(self, expr):
+        lhs_code = self._print(expr.lhs)
+        rhs_code = self._print(expr.rhs)
+        return ("{0}!={1}").format(lhs_code, rhs_code)
```

Comment:
This doesn't look right.  The relational might not be Ne, it could also be `<` or `<=`.